geo-types/src/geometry/geometry_collection.rs

```diff
@@ -248,78 +246,101 @@ impl<'a, T: CoordNum> GeometryCollection<T> {
 }
 
 #[cfg(any(feature = "approx", test))]
-impl<T> RelativeEq for GeometryCollection<T>
-where
-    T: AbsDiffEq<Epsilon = T> + CoordNum + RelativeEq,
-{
-    #[inline]
-    fn default_max_relative() -> Self::Epsilon {
-        T::default_max_relative()
-    }
-
-    /// Equality assertion within a relative limit.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use geo_types::{GeometryCollection, point};
-    ///
-    /// let a = GeometryCollection::new_from(vec![point![x: 1.0, y: 2.0].into()]);
-    /// let b = GeometryCollection::new_from(vec![point![x: 1.0, y: 2.01].into()]);
-    ///
-    /// approx::assert_relative_eq!(a, b, max_relative=0.1);
-    /// approx::assert_relative_ne!(a, b, max_relative=0.0001);
-    /// ```
-    #[inline]
-    fn relative_eq(
-        &self,
-        other: &Self,
-        epsilon: Self::Epsilon,
-        max_relative: Self::Epsilon,
-    ) -> bool {
-        if self.0.len() != other.0.len() {
-            return false;
+mod approx_integration {
+    use super::*;
+    use approx::{AbsDiffEq, RelativeEq, UlpsEq};
+
+    impl<T> RelativeEq for GeometryCollection<T>
+    where
+        T: CoordNum + RelativeEq<Epsilon = T>,
```

Comment:
Another no-op simplification in the type constraints: We don't need to specify `AbsDiffEq` **and** `RelativeEq` because `RelativeEq: AbsDiffEq`, so just `RelativeEq` is sufficient.

Reply:
Nice!